fsharp-backend/src/BackendOnlyStdLib/LibDarkInternal.fs

```diff
@@ -765,6 +765,85 @@ human-readable data."
       deprecated = NotDeprecated }
 
 
+    { name = fn "DarkInternal" "startStaticAssetDeploy" 0
+      parameters = [ Param.make "username" TStr ""; Param.make "canvasID" TUuid "" ]
+      returnType = TResult(TStr, TStr)
+      description =
+        "Records an in-progress static asset deployment, returning the deployHash"
+      fn =
+        internalFn (function
+          | _, [ DStr username; DUuid canvasID ] ->
+            uply {
+              match! Account.getUser (UserName.create username) with
```

Comment:
I think we don't need the user anymore?

Reply:
We currently use it during the INSERT of a `static_asset_deploys`, in the `uploadedBy` column ("uploadedBy", Sql.uuid user.id`